images/macos/templates/macOS-11.pkr.hcl

```diff
@@ -0,0 +1,242 @@
+    ]
+    execute_command  = "chmod +x {{ .Path }}; source $HOME/.bash_profile; {{ .Vars }} pwsh -f {{ .Path }}"
+  }
+  provisioner "shell" {
+    script = "./provision/core/reboot.sh"
+    execute_command  = "chmod +x {{ .Path }}; source $HOME/.bash_profile; sudo {{ .Vars }} {{ .Path }}"
+    expect_disconnect = true
+  }
+  provisioner "shell" {
+    scripts = [
+                "./provision/core/commonutils.sh",
+                "./provision/core/swiftlint.sh",
+                "./provision/core/openjdk.sh",
+                "./provision/core/php.sh",
+                "./provision/core/aws.sh",
+                "./provision/core/rust.sh",
+                "./provision/core/gcc.sh",
+                "./provision/core/haskell.sh",
+                "./provision/core/stack.sh",
+                "./provision/core/cocoapods.sh",
+                "./provision/core/android-toolsets.sh",
+                "./provision/core/xamarin.sh",
+                "./provision/core/vsmac.sh",
+                "./provision/core/nvm.sh",
+                "./provision/core/apache.sh",
+                "./provision/core/nginx.sh",
+                "./provision/core/postgresql.sh",
+                "./provision/core/mongodb.sh",
+                "./provision/core/audiodevice.sh",
+                "./provision/core/vcpkg.sh",
+                "./provision/core/miniconda.sh",
+                "./provision/core/chrome.sh",
+                "./provision/core/edge.sh",
+                "./provision/core/firefox.sh",
+                "./provision/core/pypy.sh",
+                "./provision/core/pipx-packages.sh",
+                "./provision/core/bicep.sh"
+    ]
+    execute_command  = "chmod +x {{ .Path }}; source $HOME/.bash_profile; {{ .Vars }} {{ .Path }}"
+  }
+  provisioner "shell" {
+    script = "./provision/core/toolset.ps1"
+    execute_command  = "chmod +x {{ .Path }}; source $HOME/.bash_profile; {{ .Vars }} pwsh -f {{ .Path }}"
+  }
+  provisioner "shell" {
+    script = "./provision/core/delete-duplicate-sims.rb"
+    execute_command  = "source $HOME/.bash_profile; ruby {{ .Path }}"
+  }
+  provisioner "shell" {
+    inline = [
+      "pwsh -File \"$HOME/image-generation/software-report/SoftwareReport.Generator.ps1\" -OutputDirectory \"$HOME/image-generation/output/software-report\" -ImageName UUID=${build.PackerRunUUID}",
+      "pwsh -File \"$HOME/image-generation/tests/RunAll-Tests.ps1\""
+    ]
+    execute_command  = "source $HOME/.bash_profile; {{ .Vars }} {{ .Path }}"
+  }
+  provisioner "file" {
+    destination = "../image-output/"
+    direction = "download"
+    source      = "./image-generation/output/"
+  }
+  provisioner "shell" {
+    scripts = [
+      "./provision/configuration/configure-hostname.sh",
+      "./provision/configuration/configure-tccdb-macos11.sh",
+      "./provision/configuration/finalize-vm.sh"
+    ]
+    execute_command  = "chmod +x {{ .Path }}; source $HOME/.bash_profile; {{ .Vars }} {{ .Path }}"
```

Comment:
We run the same command `chmod +x {{ .Path }}; source $HOME/.bash_profile; {{ .Vars }} {{ .Path }}` in every `execute_command`. Is there any way to set default value for `execute_command`?
Or may be we can move this line to variables in reuse?
Especially, I am concerned by repeating `source $HOME/.bash_profile;`

Reply:
Unfortunately, there is no way to change this behaviour